db/migrate/20170613144708_create_openshift_image.rb

```diff
@@ -0,0 +1,12 @@
+class CreateOpenshiftImage < ActiveRecord::Migration[5.0]
+  class ContainerImage < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  def change
+    add_column :container_images, :type, :string
+    ContainerImage.update_all(:type => "OpenshiftContainerImage")
+    ContainerImage.where(:command => nil).update_all(:type => "BasicContainerImage")
```

Comment:
- These 2 data migration bits cannot be in a `change` method as they are not reversible.  Please split this into an up and a down.
- Please wrap data migrations in a say_with_time like the other data migrations
- Data migrations *must* have a migration spec